snapcraft.yaml

```diff
@@ -19,6 +19,8 @@ parts:
     requirements: python/requirements.txt
     plugin: python
     python-version: python3
+    stage-packages:
+        - python3-dbus
```

Comment:
This part is only in place to install python3. Why is python3-dbus staged here?

Reply:
I thought the same, it was in the other part and I had errors. Here it works fine.